src/Optimization/hiopHessianLowRank.cpp

```diff
@@ -328,14 +340,14 @@ bool hiopHessianLowRank::update(const hiopIterate& it_curr, const hiopVector& gr
     }
 
     //save this stuff for next update
-    _it_prev->copyFrom(it_curr);  _grad_f_prev->copyFrom(grad_f_curr); 
+    _it_prev->copyFrom(it_curr);  _grad_f_prev->copyFrom(grad_f_curr_); 
```

Comment:
can you please clean up the code a bit. Enforce coding style guidelines, remove variables starting with "_", use `nullptr`. I can fill an issue with these and we can address these later, after we merge this, if this PR urgent.

Reply:
Well, it seems too many lines need to be updated.
I just opened a new issue and I will address all the code style in a new PR, so we can see the changes in code style easily in that PR.